client/lib/upgrades/actions/free-trials.js

```diff
@@ -1,48 +0,0 @@
-/** @format */
```

Comment:
why is this file deleted?

Reply:
We don't do free trials anymore. It's not used anywhere.